### PR TITLE
Rsdk 6599 Add non-erroring GetAccuracy implementations to FIX RC Card polling of API

### PR DESCRIPTION
**Pull request by randhid**
The RC card currently polls GetAccuracy, sees an error and refuses to show up since it is unimplemented in most drivers.
This PR adds a helper function that will not error, but populates the drivers with the values that will indicate to the RC card to hide the CompassHeading, HDOP, VDOP, and NMEAFIX from the card for drivers that cannot report them

### CODE REVIEW

File: viam-rand-s-mac-main.json [hunk 1]
```diff
@@ -0,0 +1 @@
+{"cloud":{"app_address":"https://app.viam.com:443","id":"50dc0a41-0f21-4186-9394-5d26d9e2b053","secret":"4603q9h23gvg93555qnuhi8br2whqg1ntdv9d6qg5215bxxh"}}
```
Comment (penguinland) [inline viam-rand-s-mac-main.json]: This isn't supposed to be in here, right? Please rotate your secret when you remove this.

File: components/movementsensor/replay/replay_test.go [hunk 2]
```diff
@@ -764,8 +764,8 @@ func TestUnimplementedFunctionAccuracy(t *testing.T) {
 	test.That(t, err, test.ShouldBeNil)
 
 	acc, err := replay.Accuracy(ctx, map[string]interface{}{})
-	test.That(t, err, test.ShouldResemble, movementsensor.ErrMethodUnimplementedAccuracy)
-	test.That(t, acc, test.ShouldBeNil)
+	test.That(t, err, test.ShouldResemble, nil)
```
Comment (penguinland) [inline components/movementsensor/replay/replay_test.go:767]: What's the difference between `test.ShouldResemble, nil` and just `test.ShouldBeNil`?

File: components/movementsensor/imuwit/imu.go [hunk 3]
```diff
@@ -202,7 +202,12 @@ func (imu *wit) Position(ctx context.Context, extra map[string]interface{}) (*ge
 
 func (imu *wit) Accuracy(ctx context.Context, extra map[string]interface{}) (*movementsensor.Accuracy, error,
 ) {
-	return nil, movementsensor.ErrMethodUnimplementedAccuracy
+	// RSDK-6389 return the compass heading from the datasheet of the witIMU if the pitch angle is less than 45 degrees
+	// and the roll angle is near zero
+	// mag projects at angles over this threshold cannot be determined because of the larger contribution of other
+	// orientations to the true compass heading
+	// return NaN for compass accuracy otherwise.
+	return movementsensor.UnimplementedAccuracies()
```
Comment (penguinland) [inline components/movementsensor/imuwit/imu.go:210]: Comment doesn't match code: we're treating it as unimplemented even if the pitch angle is low. Was the comment supposed to be a TODO of things to add in the future?

File: components/movementsensor/imuvectornav/imu.go [hunk 4]
```diff
@@ -280,7 +280,9 @@ func (vn *vectornav) Position(ctx context.Context, extra map[string]interface{})
 }
 
 func (vn *vectornav) Accuracy(ctx context.Context, extra map[string]interface{}) (*movementsensor.Accuracy, error) {
-	return nil, movementsensor.ErrMethodUnimplementedAccuracy
+	// RSDK-6389 check the vectornav's datasheet to determine what is best to return from the vector nav.
+	// can be done in a seprate ticket from the one mentioned in this comment.
```
Comment (penguinland) [inline components/movementsensor/imuvectornav/imu.go:284]: Is this comment a TODO? I don't get what it means if it's not.

File: viam-rand-s-mac-main.json (hunk 1 above)
Comment (randhid) [reply viam-rand-s-mac-main.json]: Yeah, didn't notice it before i pushed, closing this, deleting the branch and rotating the secret in the relocation it's in

File: components/movementsensor/dualgps/dualgps.go [hunk 5]
```diff
@@ -264,7 +264,12 @@ func (dg *dualGPS) Orientation(ctx context.Context, extra map[string]interface{}
 }
 
 func (dg *dualGPS) Accuracy(ctx context.Context, extra map[string]interface{}) (*movementsensor.Accuracy, error) {
-	return nil, movementsensor.ErrMethodUnimplementedAccuracy
+	// RSDK-6389: for this driver, find the highest value VDOP and HDOP to show the worst accuracy in position
+	// check the fix and return the compass error using a calculation based on the fix and variation
+	// of the two positions.
+	// return the NMEA Fix that appears from either gps in this order: (worst) 0, 1, 2, 5, 4 (best), or -1 if
+	// it is not any of these (other nmea GGA fixes indicate simulated or deadreckoning gps devices)
+	return movementsensor.UnimplementedAccuracies()
```
Comment (penguinland) [inline components/movementsensor/dualgps/dualgps.go:272]: Same confusion re: comment not matching code here. If a comment is meant as a TODO, I'd start it off with `// TODO[RSDK-6389]:` to make that clear.

**Review by penguinland — COMMENTED** [review]
...oh, this PR is closed. Well, my main feedback so far is to mark which comments are TODOs, to separate them from the unmarked FYIs.

**Pull request closed without merging**